inst/include/cpp11/protect.hpp

```diff
@@ -54,20 +54,34 @@ inline void set_option(SEXP name, SEXP value) {
   SETCAR(opt, value);
 }
 
-inline Rboolean& get_should_unwind_protect() {
+inline Rboolean* setup_should_unwind_protect() {
```

Comment:
I can continue to make this return a reference rather than a pointer, if anyone has any preference. It seemed a little easier for me to understand if this was a pointer.

Reply:
I don't mind it being a pointer.